### PR TITLE
added sql export for database (with content in lookup tables

### PR DESCRIPTION
**Pull request by MotNosniktaw**

### CODE REVIEW

File: Guitars_2019-08-05.sql [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+DROP TABLE IF EXISTS `guitars`;
+
+CREATE TABLE `guitars` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `brand` int(11) unsigned NOT NULL,
+  `model` varchar(255) NOT NULL DEFAULT '',
+  `year` year(4) DEFAULT NULL,
+  `type` int(11) unsigned NOT NULL,
+  `country` int(11) unsigned NOT NULL,
+  `LH or RH` varchar(255) NOT NULL DEFAULT '',
```
Comment (FelixLetheren) [inline Guitars_2019-08-05.sql]: Could be ENUM instead of VARCHAR as only 2 options.

**Review by FelixLetheren — CHANGES_REQUESTED** [review]
All Seems very fancy and nicely done. Just 1 suggestion

File: Guitars_2019-08-05.sql [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+
+# Dump of table guitars
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `guitars`;
+
+CREATE TABLE `guitars` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `brand` int(11) unsigned NOT NULL,
```
Comment (sjr-mayden) [inline Guitars_2019-08-05.sql]: Something like brandID here would probably be a better indicator of what this field is

File: Guitars_2019-08-05.sql [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+  `model` varchar(255) NOT NULL DEFAULT '',
+  `year` year(4) DEFAULT NULL,
+  `type` int(11) unsigned NOT NULL,
+  `country` int(11) unsigned NOT NULL,
+  `LH or RH` enum('LH','RH') NOT NULL,
+  `value` int(11) DEFAULT NULL,
+  `serialCode` int(11) DEFAULT NULL,
```
Comment (sjr-mayden) [inline Guitars_2019-08-05.sql]: Are serial codes just numbers? Are they never ever going to contain any letters?

File: Guitars_2019-08-05.sql [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `type` varchar(255) NOT NULL DEFAULT '',
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+LOCK TABLES `type` WRITE;
+/*!40000 ALTER TABLE `type` DISABLE KEYS */;
+
+INSERT INTO `type` (`id`, `type`)
+VALUES
```
Comment (sjr-mayden) [inline Guitars_2019-08-05.sql:136]: This feels like more than one field should be used, maybe electrical/acoustic as an enum, a number of strings and then an enum range (tenor etc.)?

**Review by sjr-mayden — CHANGES_REQUESTED** [review]

**Pull request merged**